### ISSUE TITLE
No data after update Watecollector Cyclus

### ISSUE THREAD

**Issue by Dutchstyler** [pippyn/Home-Assistant-Sensor-Afvalbeheer#507]
After the last update, I noticed that everything has the state unknown.
Had the old config, so I have changed it to the new way, direct in the configfile, without succes.
First thought that i had a typo so i just copies from the github page and changed the values. Without succes also.

The logfile says:
Bron: custom_components/afvalbeheer/API.py:995
Error occurred while fetching data: JSONDecodeError('Expecting value: line 1 column 1 (char 0)')

My config:
```
#Afvalbeheer
afvalbeheer:
  wastecollector: Cyclus
  resources:
    - restafval
    - gft
    - papier
    - pmd
  postcode: 2743EW
  streetnumber: 16
  upcomingsensor: 1 # (optional)
  dateformat: "%d-%m" # (optional)
  dateonly: 0 # (optional)
  name: "" # (optional)
  nameprefix: 1 # (optional)
  builtinicons: 1 # (optional)
  dutch: 1 # (optional)
  printwastetypes: 1
  updateinterval: 12
```

**Comment by YvoTuk** [comment]
Same problem here, also using cyclus.

**Comment by Jrrsh** [comment]
Same problem here, using Cyclus, postcode 2922

**Comment by Naappie** [comment]
Same here postcode 3417

**Comment by pippyn** [comment]
Cyclus changed their API endpoint url. Fixed in v5.4.1

**Comment by Naappie** [comment]
This fixed it for me! Thanks for the fast response!

**Comment by Dutchstyler** [comment]
> Cyclus changed their API endpoint url. Fixed in v5.4.1

Thank you a lot. This fixed it. Thanks for the fast update.
